src/Promenade.Service/Interfaces/Repositories/ISegmentTextRepository.cs

```diff
@@ -0,0 +1,11 @@
+﻿
+using Ocuda.Promenade.Models.Entities;
+
+namespace Ocuda.Promenade.Service.Interfaces.Repositories
+{
+    public interface ISegmentTextRepository : IGenericRepository<SegmentText, int>
+    {
+        public SegmentText GetSegmentTextBySegmentId(int segmentId);
```

Comment:
This method is not used. Remove from interface and implementation.

Reply:
Removed